### ISSUE TITLE
where is the code?

### ISSUE THREAD

**Issue by Karry-Wu** [yiwangtz/MENet#3]

**Comment by moothes** [comment]
I just noticed that the code was removed by the authors.
As a top conference, I think authors should make their code publicly available.
I copied the code last year and adapted its implementation to my [benchmark](https://github.com/moothes/SALOD).
You can train this network in my benchmark, but I can't guarantee that you'll achieve similar results to those reported in the paper.
You may need to modify the training settings based on the paper, including the optimizer and loss function.

**Comment by Karry-Wu** [comment]
Thank you for your contribution, I will follow your work and give it a star.

**Comment by HangSS** [comment]
I wonder why the author deleted the code? What's the point of disclosing only the results?
